install_sct

```diff
@@ -537,6 +537,10 @@ else
   # Not a package
   print info "Using requirements.txt (git installation)"
   pip install -r requirements.txt
+  yes | pip uninstall tensorflow-tensorboard
+  yes | pip uninstall tensorboard
+  pip install tensorboard
```

Comment:
Also, do you know if reinstalling `tensorboard` is absolutely necessary?

Reply:
Yes actually, I think uninstalling `tensorflow-tensorboard` leaves some dependencies for tensorboard, but they are outdated.. Maybe an upgrade would be enough?